apps/financial-aid/backend/src/app/modules/personalTaxReturn/personalTaxReturn.service.ts

```diff
@@ -11,46 +11,86 @@ export class PersonalTaxReturnService {
     private fileService: FileService,
   ) {}
 
-  private async callPersonalTaxReturn(nationalId: string, year: number) {
-    return await this.personalTaxReturnApi
-      .personalTaxReturnInPdf(nationalId, year.toString())
-      .catch(() => {
-        return { success: false, content: '' }
-      })
+  async directTaxPayments(nationalId: string) {
+    try {
+      const directTaxPayments = await this.personalTaxReturnApi.directTaxPayments(
+        nationalId,
+        this.createPeriod(3),
+        this.createPeriod(1),
+      )
+
+      return {
+        directTaxPayments: directTaxPayments.salaryBreakdown.map((salary) => {
+          return {
+            totalSalary: salary.salaryTotal,
+            payerNationalId: salary.payerNationalId.toString(),
+            personalAllowance: salary.personalAllowance,
+            withheldAtSource: salary.salaryWithheldAtSource,
+            month: salary.period,
+            year: salary.year,
+          }
+        }),
+      }
+    } catch {
+      return undefined
+    }
   }
 
   async personalTaxReturn(nationalId: string, folder: string) {
-    let year = new Date().getFullYear() - 1
+    try {
+      let year = new Date().getFullYear() - 1
```

Comment:
I know I have commented on this before, I just feel like this const is very confusing, maybe changeableYear?